readme.md

```diff
@@ -70,19 +70,25 @@ export class AppModule {}
 
 ### Sass
 
-Include the Kirby global styles in your app:
+Include the Kirby global styles in your app, e.g., in `src/styles.scss` :
 
-- Eg. in `src/styles.scss`:
+```css
+@import '~@kirbydesign/designsystem/scss/global-styles';
+```
+
+In each `.scss` file where you need to access the Sass utility functions from Kirby (e.g. [colors][kirby.cookbook.colors] or [fonts][kirby.cookbook.fonts]) you must import the scss utilities:
 
-  ```css
-  @import '~@kirbydesign/designsystem/scss/global-styles';
-  ```
+```css
+@import '~@kirbydesign/designsystem/scss/utils';
+```
 
-  In each `.scss` file where you need to access the Sass utility functions from Kirby (e.g. [colors][kirby.cookbook.colors] or [fonts][kirby.cookbook.fonts]) you must import the scss utilities:
+Kirby also provides a generic print stylesheet. It includes the basics. You most likely have to add local print styles specific to your app as well.
```

Comment:
Consider adding a heading for easy skimming and to make the reader aware that this part is wholly optional: 
```suggestion
#### Generic Print Styles (Optional)
Kirby also provides a generic print stylesheet. It includes the basics. You most likely have to add local print styles specific to your app as well.
```

Reply:
I actually started out with a heading, but removed it. We can add it again. The word _optional_ is a good addition :)